changelogs/internal/newsfragments/2045.clarification

```diff
@@ -0,0 +1 @@
+Clarify that v1.0 of Matrix was a symbolic release prior to the current global versionins system.
```

Comment:
I am not sure where to add the changelog, it doesn't seem that there is a section for the homepage of the spec.